tdrs-backend/tdpservice/parsers/test/test_parse.py

```diff
@@ -1726,3 +1726,87 @@ def test_parse_tribal_section_4_bad_quarter(tribal_section_4_bad_quarter, dfs):
         "representing the Calendar Year and Quarter formatted as YYYYQ"
 
     Tribal_TANF_T7.objects.count() == 0
+
+@pytest.mark.django_db()
+def test_parse_t3_cat2_invalid_citizenship(t3_cat2_invalid_citizenship_file, dfs):
+    """Test parsing a TANF T3 record with an invalid CITIZENSHIP_STATUS."""
+    dfs.datafile = t3_cat2_invalid_citizenship_file
+    t3_cat2_invalid_citizenship_file.year = 2021
+    t3_cat2_invalid_citizenship_file.quarter = 'Q1'
+    dfs.save()
+
+    parse.parse_datafile(t3_cat2_invalid_citizenship_file, dfs)
+
+    parser_errors = ParserError.objects.filter(file=t3_cat2_invalid_citizenship_file).order_by("pk")
+
+    assert parser_errors.count() == 2
+
+    citizenship_status_1_error = parser_errors[0]
+    citizenship_status_2_error = parser_errors[1]
+
+    assert citizenship_status_1_error.error_message == "T3: 0 is not in [1, 2, 9]."
+    assert citizenship_status_2_error.error_message == "T3: 0 is not in [1, 2, 9]."
```

Comment:
so if the error messages are the same, I suggest you don't use indexing on the error message, as the index might change, instead assert: ```"T3: 0 is not in [1, 2, 9]." is in error_message for error_message in citizenship_status_1_error.error_message```